applications/smart-distancing/tools/environment_score.py

```diff
@@ -0,0 +1,51 @@
+"""
+Set of functions related to calculating environment score for physical distancing is implemented here.
+"""
+import numpy as np
+
+# TODO: consider these as a parameter; In the future, depend on the project, they may be moved at a separate config file
+MAX_ACCEPTABLE_CAPACITY = 20  # The maximum number of people that can stand as far away from other people as possible
```

Comment:
I can't understand what does this variable stand for? maybe a little bit description would be helpful.